ci/linux/run.sh

```diff
@@ -241,6 +241,15 @@ fi
 if [ $UI_TESTS -eq 1 ]; then
     clean_uitest
 
+    ensure_named_command "npm install rollup (global)" \
+        "cd \"$ROOT/src/Samples/Common\" \
+           &&  npm install -g rollup"
+
+    ensure_named_command "npm build" \
+        "cd \"$ROOT/src/Samples/Common\" \
+            && npm ci --cache \"$ROOT/.npm\" --prefer-offline \
+            && rollup -c"
```

Comment:
oooou, I think you could do `npm run rollup -c` or `yarn rollup -c`. Or copy the MSBuild magic that we have in Dotvvm.Framwork.csproj